.github/workflows/common-app-build.yml

```diff
@@ -18,6 +18,14 @@ on:
         required: false
         type: string
         default: 0.14.0
+      install-package:
+        required: false
```

Comment:
nebude lepsie to nechat ako default, ze sa to instaluje? nech nemame taketo problemy do buducna a nemusime riesit, ktore repo ide cez parallel a ktore nie

Reply:
vyskúšam, že ako sa to bude správať + default command skúsim zmeniť na empty